### PR TITLE
Send surveys by email

### PR DESCRIPTION
**Pull request by carlosfaria94**
Refers to #61 

Work in Progress

### CODE REVIEW

**Comment by carlosfaria94** [comment]
@johnytiago I think it's done. Can you review?

File: server/routes/survey/handlers.js [hunk 1]
```diff
@@ -13,7 +13,8 @@ exports.submit = {
   validate: {
     params: {
       redeemCode: Joi.string().required().description('redeem code')
-    }
+    },
+    payload: {}
```
Comment (johnytiago) [inline server/routes/survey/handlers.js]: If you don't use the `payload` why are you declaring it?

File: server/routes/survey/handlers.js [hunk 2]
```diff
@@ -84,3 +85,30 @@ exports.getSessionProcessedResponses = {
   },
   description: 'Get processed responses of a session'
 }
+
+exports.checkIn = {
+  tags: ['api', 'survey'],
+  auth: {
+    strategies: ['default', 'backup'],
+    scope: ['admin']
```
Comment (johnytiago) [inline server/routes/survey/handlers.js]: Please add `team` as an accepted `scope`

File: server/routes/survey/handlers.js (hunk 1 above)
Comment (johnytiago) [reply server/routes/survey/handlers.js]: Ahh, ok! Will re-add it then! ;)

**Pull request merged**